Apps/Common/TextureProperties.C

```diff
@@ -0,0 +1,128 @@
+// $Id$
+//==============================================================================
+//!
+//! \file TextureProperties.C
+//!
+//! \date Jan 3 2020
+//!
+//! \author Arne Morten Kvarving / SINTEF
+//!
+//! \brief Isotropic linear elastic material model defined through a texture map.
```

Comment:
This description is now perhaps to specific. I can be used for any spatially distributed property (more like a field).

Reply:
yeah. I did the .h, forgot the .C file.